tests/test_cli.py

```diff
@@ -382,18 +382,20 @@ def invoke_gc(*args):
     [
         # MLServer is not supported in Windows yet, so let's skip this test in that case.
         # https://github.com/SeldonIO/MLServer/issues/361
-        pytest.param(
-            True,
-            marks=pytest.mark.skipif(
-                os.name == "nt", reason="MLServer is not supported in Windows"
-            ),
-        ),
+        # Re-enable enable_mlserver=True once inference parameters change is
+        # updated to MLServer repo
+        # pytest.param(
+        #     True,
+        #     marks=pytest.mark.skipif(
+        #         os.name == "nt", reason="MLServer is not supported in Windows"
+        #     ),
+        # ),
```

Comment:
Can we continue to test MLserver for use cases where `params` are missing from the signature? I.e. does this test fail if these lines are uncommented? If so, why?

Reply:
Yes it fails, but because I refactored scoring_server/__init__.py file and changed the function names, MLServer import old functions and copied our logics, so it fails.

Reply:
Can we undo the refactor so that we remain compatible? If MLServer relies on `scoring_server/init.py`, then I think we should decorate the methods that it depends on as @developer_stable and make sure we don't make backwards-incompatible changes to those methods (e.g. don't remove them)

Reply:
Instead of undoing the refactor, could we still keep the original function and mark it as deprecated https://github.com/mlflow/mlflow/pull/8563/commits/cf2f9d2ee544c9eb1916f2abb19fdc8883041f35 ?